custom/opm/reports.py

```diff
@@ -1112,7 +1116,7 @@ def rows():
                 for method, __, __, denom in self.model.method_map:
                     value = getattr(awc, method)
                     row.append(value)
-                    if denom != 'no_denom':
+                    if denom != 'no_denom' and denom != 'one':
```

Comment:
It does sum it up in the totals, but OPM has asked if we can display percentage in that totals. With current setup of `no_denom` it's not possible to display percentage in totals but not in individual rows. So I made this special denom to do so.